beta/src/components/MDX/LiveRunner/LivePreview.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {useState, useMemo} from 'react';
+import {Runner, importCode, Scope} from 'react-runner';
+import * as UseImmer from 'use-immer';
```

Comment:
It's not super great that we have to load everything we use in every sandbox. We might want to have "heavier" examples in the future with third party libs. They shouldn't bloat the initial editor. I guess having a list of them here is probably fine though. Maybe as dynamic imports and somehow lazy-loaded?

Reply:
Yes lazy loading should definitely work, and we can still keep the `package.json` file and analyse the dependency, right now I just ignored that file